tools/bridge/docker/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2.4'
+services:
+
+  validator:
```

Comment:
```suggestion
  bridge_validator:
```

Reply:
Strong suggestion :wink:

Reply:
The suggested project name is tlbc-bridge, so the container name will be tlbc-bridge_validator_1